src/app/functionality/check_points_logic.py

```
@@ -1,16 +1,23 @@
 from src.app.functionality.run_activy import run_activy
+from src.database.DbHandler import DbHandler
 
 
-def check_points_logic(accounts, db_handler):
-    device = run_activy()
+def check_points_logic(accounts):
+    try:
+        device = run_activy()
+    except Exception as e:
+        print(f"Error starting AVD: {e}")
+        raise ValueError("Failed to start AVD. Please check your configuration.")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)